src/components/Journal/JournalEntryForm/journalEntryLineItem.scss

```diff
@@ -0,0 +1,44 @@
+.Layer__JournalEntryForm__LineItem {
+  display: flex;
+  align-items: flex-end;
+  gap: var(--spacing-xs);
+
+  .Layer__JournalEntryForm__Field {
+    &--accountName {
+      flex: 1 1 12rem;
+      max-width: 20rem;
+    }
+
+    &--amount {
+      flex: 0 1 12rem;
+      max-width: 12rem;
+    }
+
+    &--tag {
+      flex: 0 0 auto;
+      min-width: 12rem;
+    }
```

Comment:
I would love to just use a grid, but I'm handling here the case where there could be multiple different tag inputs.

Reply:
Okay I've hacked a new way to do this: using data properties to specify the number of tag columns I do think it's a marginal improvement in that we don't have to have sizing for every individual field. Not sure it's worth it, but keeping it for now.